jupyterhub/orm.py

```diff
@@ -220,6 +220,15 @@ def find(cls, db, name):
         """
         return db.query(cls).filter(cls.name == name).first()
 
+    @classmethod
+    def find_by_id(cls, db, id):
```

Comment:
Note that we won't need this if we join the user with the servers when we fetch the servers from the DB. Maybe that's already done though with this?

```python
_orm_spawners = relationship(
    "Spawner", backref="user", cascade="all, delete-orphan"
)
```